R/download.R

```diff
@@ -1,9 +1,3 @@
-# Kaiaulu - https://github.com/sailuh/kaiaulu
-#
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
-
```

Comment:
Not sure why this was removed... but I guess I will just re-add after the merge.